com.unity.multiplayer.mlapi/Runtime/Core/SnapshotRTT.cs

```diff
@@ -0,0 +1,94 @@
+using System;
+using UnityEngine;
+
+namespace MLAPI
+{
+    internal class ClientRtt
```

Comment:
Could be a struct instead of a class?

Reply:
Agreed this should be a struct

Reply:
sorry, I'm sure this was changed to struct, apparently there was a snafu. Will fix.